.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -168,12 +168,6 @@ jobs:
           IMAGE_NAME=${{ steps.resolve-image.outputs.image }}
           echo "Scanning $IMAGE_NAME"
 
-          SEVERITY_OPTION=""
-          # Report only higher vulnerabilities if not a pull request
-          if [ "${{ fromJson(inputs.github).event_name }}" != "pull_request" ]; then
-            SEVERITY_OPTION="--severity CRITICAL,HIGH"
-          fi
```

Comment:
you're deleting this, not adding! sorry for raving madly before

Reply:
NP! Please check the description: _The default is all severities if you don't provide the --severity option._